R/nma.R

```diff
@@ -2888,6 +2934,9 @@ make_nma_formula <- function(regression,
       nma_formula <- regression
     }
 
+    # Remove any main effect of .mu if baseline risk regression used
+    if (".mu" %in% all.vars(regression)) nma_formula <- update.formula(nma_formula, ~. - .mu)
```

Comment:
Maybe abort here to prevent confusion?

```suggestion
    if (".mu" %in% attr(terms(regression), "term.labels")) {
      abort("`.mu` is not allowed as main effect in the `regression`.")
    }
```

Reply:
I silently removed any main effects, because this is how main effects of `.trt` and `.trtclass` are handled too. The "main effects" of `.mu` are still in the model in a way - they are the intercepts. The proposed change gives an error for `~.mu*.trt`, which would be nice to silently correct (the same way that the main effects of `.trt` are handled in this case). 

If we can distinguish between `~.mu*.trt` and `~.mu + ...` then we could add a warning that the main effects are being removed for the latter? This would probably need to go in `nma()` rather than `make_nma_formula()`, to avoid this error firing when `make_nma_formula()` is called in predict etc.